crates/re_space_view_spatial/benches/bench_points.rs

```diff
@@ -51,88 +51,135 @@ fn bench_points(c: &mut criterion::Criterion) {
     };
 
     let latest_at = LatestAtQuery::latest(timeline);
+    let at = latest_at.at;
+    let latest_at = re_query_cache::AnyQuery::from(latest_at);
     let annotations = Annotations::missing();
 
     {
         let mut group = c.benchmark_group("Points3D");
         group.bench_function("query_archetype", |b| {
             b.iter(|| {
-                let arch_view =
-                    re_query::query_archetype::<Points3D>(&store, &latest_at, &ent_path).unwrap();
-                assert_eq!(arch_view.num_instances(), NUM_POINTS);
-                arch_view
+                re_query_cache::query_archetype_pov1_comp5::<
+                    Points3D,
+                    Position3D,
+                    Color,
+                    Radius,
+                    Text,
+                    re_types::components::KeypointId,
+                    re_types::components::ClassId,
+                    _,
+                >(
+                    true,
+                    &store,
+                    &latest_at,
+                    &ent_path,
+                    |(_, keys, _, _, _, _, _, _)| {
+                        assert_eq!(keys.as_slice().len(), NUM_POINTS);
+                    },
+                )
+                .unwrap();
```

Comment:
a bit hard to read: What is `pov1` and what does the `true` parameter do

more local: strange selection of types that isn't under a `use`

Reply:
I've added a comment for the `true` param and imported `KeypointId` & `ClassId`.

I can't really think of any way  to make the `povN_compM` clearer that what the docstring of the method already says :/